tests/unit/amo/components/TestAddonBadges.js

```diff
@@ -106,4 +106,24 @@ describe(__filename, () => {
 
     expect(root.find(Badge)).toHaveLength(0);
   });
+
+  it('displays a badge when the addon is not a web extension', () => {
```

Comment:
There should be an extra test here for it not being a web extension but also not being an `addonType` of `ADDON_TYPE_EXTENSION`.

Reply:
ah good catch, just added it!